test/test_sbml_conversion.py

```diff
@@ -26,24 +29,24 @@ def runTest(self):
             x0 = list(model.getParameters())
             df = objective.check_grad(
                 x0,
-                eps=1e-5,
+                eps=1e-3,
                 verbosity=0,
                 mode=pypesto.objective.constants.MODE_FUN
             )
             print("relative errors MODE_FUN: ", df.rel_err.values)
             print("absolute errors MODE_FUN: ", df.abs_err.values)
-            self.assertTrue(np.all(df.rel_err.values < 1e-2))
-            self.assertTrue(np.all(df.abs_err.values < 1e-1))
+            self.assertTrue(np.all((df.rel_err.values < RTOL) |
+                                   (df.abs_err.values < ATOL)))
             df = objective.check_grad(
                 x0,
-                eps=1e-5,
+                eps=1e-3,
```

Comment:
so the FD interval was too short to get reliable approximations? looks good now.

Reply:
yup